src/styles.scss

```diff
@@ -9,3 +9,14 @@ body {
   margin: 0;
   font-family: Roboto, "Helvetica Neue", sans-serif;
 }
+
+.mat-tab-body-content {
+  overflow: hidden !important;
+}
+
+button:focus {
+  outline: none !important;
+}
+
+/* Importing Bootstrap SCSS file. */
+@import "~bootstrap/dist/css/bootstrap";
```

Comment:
Typically, you would want to put this at the top of the file.

Reply:
I'm not sure why, but this is where it was added automatically with ng add